biodatasets/quaero/quaero.py

```diff
@@ -0,0 +1,280 @@
+    """TODO: Short description of my dataset."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+    DEFAULT_CONFIG_NAME = "quaero_source"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="quaero_source",
+            version=SOURCE_VERSION,
+            description="QUAERO source schema",
+            schema="source",
+            subset_id="quaero",
+        ),
+        BigBioConfig(
+            name="quaero_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="QUAERO simplified BigBio schema",
+            schema="bigbio_kb",
+            subset_id="quaero",
+        ),
+    ]
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "entities": [
+			            {
+			                "id": datasets.Value("string"),
+			                "type": datasets.Value("string"),
+			                "text": datasets.Sequence(datasets.Value("string")),
+			                "offsets": datasets.Sequence([datasets.Value("int32")]),
+			                "normalized": [
+			                    {
+			                        "db_name": datasets.Value("string"),
+			                        "db_id": datasets.Value("string"),
+			                    }
+			                ],
+			            }
+			        ],
+			        "passages": [
+			            {
+			                "id": datasets.Value("string"),
+			                "type": datasets.Value("string"),
+			                "text": datasets.Sequence(datasets.Value("string")),
+			                "offsets": datasets.Sequence([datasets.Value("int32")]),
+			            }
+        ],
+                      }
+            )
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+            
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _get_bioc_entity(self, span, doc_text, db_id_key=None):
+        """Parse BioC entity annotation.
+        Parameters
+        ----------
+        span : BioCAnnotation
+            BioC entity annotation
+        doc_text : string
+            document text, required to construct text spans
+        db_id_key : str, optional
+            database name used for normalization, by default "MESH"
+        Returns
+        -------
+        dict
+            entity information
+        """
+        offsets = [(loc.offset, loc.offset + loc.length) for loc in span.locations]
+        texts = [doc_text[i:j] for i, j in offsets]
```

Comment:
I am sorry but we have recently updated the way we handle bioc files. 
Could you please change this following [this example](https://github.com/bigscience-workshop/biomedical/blob/master/examples/bc5cdr.py#L220)?